### PR TITLE
schemas: replace applicable directions to direction for buffer stops

### LINKED ISSUE #3391: Remove applicable_directions field in buffer stops
### Description and goal

A buffer stop doesn't need a direction.

### Acceptance criteria

- [ ] Adapt railjson

### PR DESCRIPTION
**Pull request by flomonster**
Linked to #3607 
close #3391

### CODE REVIEW

File: core/examples/generated/infra.json
Comment (multun) [inline core/examples/generated/infra.json:1]: it adding this file intended?

File: core/examples/generated/simulation.json
Comment (multun) [inline core/examples/generated/simulation.json:1]: is this intended?

**Review by Castavo — DISMISSED** [review]

File: core/examples/generated/lib/railjson_generator/schema/infra/waypoint.py [hunk 1]
```diff
@@ -1,51 +1,46 @@
...
-
-
-@dataclass
-class BufferStop(Waypoint):
-    waypoint_type: str = "buffer_stop"
-    label: str = field(default_factory=_buffer_stop_id)
-
-    _INDEX = 0
+    
+    def get_direction(self, track) -> Direction:
```
Comment (Tguisnet) [inline core/examples/generated/lib/railjson_generator/schema/infra/waypoint.py:40]: I don't understand this function

File: core/examples/generated/lib/railjson_generator/infra_builder.py [hunk 2]
```diff
@@ -1,17 +1,14 @@
 from dataclasses import dataclass, field
 from typing import Mapping, Tuple, Type
 
+from .schema.infra.direction import Direction
```
Comment (Tguisnet) [inline core/examples/generated/lib/railjson_generator/infra_builder.py]: I think this is not use

**Review by Tguisnet — CHANGES_REQUESTED** [review]

**Review by Tguisnet — APPROVED** [review]

**Pull request merged**